components/ws-manager/pkg/manager/testdata/status_cannotPull_004_CREATING00.golden

```diff
@@ -13,12 +13,6 @@
             "workspace_image": "eu.gcr.io/gitpod-dev/workspace-images:01abb31dd76a1e2885ea1d5c3b06a4ead10ae362ee21348428835d0c8b286e3a",
             "deprecated_ide_image": "eu.gcr.io/gitpod-core-dev/build/ide/code:commit-0941a0805dc3c7345c45bd926317eaf045d4b7fb",
             "url": "https://silver-dormouse-is733prg.ws-us14.gitpod.io",
-            "exposed_ports": [
-                {
-                    "port": 3000,
-                    "url": "https://3000-silver-dormouse-is733prg.ws-us14.gitpod.io"
-                }
-            ],
```

Comment:
Why do all the exposed_ports go away in the status?

Reply:
Technically it's correct, no? We're still in creating, so the pod does not expose anything yet. Still, the question is if this change breaks sth.